src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -4576,14 +4564,14 @@ void SetUpSysSizingArrays(EnergyPlusData &state)
         CalcSysSizing(AirLoopNum).FloorAreaOnAirLoopCooled = 0.0;
         CalcSysSizing(AirLoopNum).FloorAreaOnAirLoopHeated = 0.0;
 
-        SysSizPeakDDNum(AirLoopNum).TimeStepAtSensCoolPk.allocate(state.dataEnvrn->TotDesDays + state.dataEnvrn->TotRunDesPersDays);
-        SysSizPeakDDNum(AirLoopNum).TimeStepAtTotCoolPk.allocate(state.dataEnvrn->TotDesDays + state.dataEnvrn->TotRunDesPersDays);
-        SysSizPeakDDNum(AirLoopNum).TimeStepAtCoolFlowPk.allocate(state.dataEnvrn->TotDesDays + state.dataEnvrn->TotRunDesPersDays);
-        SysSizPeakDDNum(AirLoopNum).TimeStepAtHeatPk.allocate(state.dataEnvrn->TotDesDays + state.dataEnvrn->TotRunDesPersDays);
-        SysSizPeakDDNum(AirLoopNum).TimeStepAtSensCoolPk = 0;
-        SysSizPeakDDNum(AirLoopNum).TimeStepAtTotCoolPk = 0;
-        SysSizPeakDDNum(AirLoopNum).TimeStepAtCoolFlowPk = 0;
-        SysSizPeakDDNum(AirLoopNum).TimeStepAtHeatPk = 0;
+        state.dataSize->SysSizPeakDDNum(AirLoopNum).TimeStepAtSensCoolPk.allocate(state.dataEnvrn->TotDesDays + state.dataEnvrn->TotRunDesPersDays);
+        state.dataSize->SysSizPeakDDNum(AirLoopNum).TimeStepAtTotCoolPk.allocate(state.dataEnvrn->TotDesDays + state.dataEnvrn->TotRunDesPersDays);
+        state.dataSize->SysSizPeakDDNum(AirLoopNum).TimeStepAtCoolFlowPk.allocate(state.dataEnvrn->TotDesDays + state.dataEnvrn->TotRunDesPersDays);
+        state.dataSize->SysSizPeakDDNum(AirLoopNum).TimeStepAtHeatPk.allocate(state.dataEnvrn->TotDesDays + state.dataEnvrn->TotRunDesPersDays);
+        state.dataSize->SysSizPeakDDNum(AirLoopNum).TimeStepAtSensCoolPk = 0;
+        state.dataSize->SysSizPeakDDNum(AirLoopNum).TimeStepAtTotCoolPk = 0;
+        state.dataSize->SysSizPeakDDNum(AirLoopNum).TimeStepAtCoolFlowPk = 0;
+        state.dataSize->SysSizPeakDDNum(AirLoopNum).TimeStepAtHeatPk = 0;
```

Comment:
Hmm, I could have referenced this. 8 array interpretations exceeds my 3 interpretation limit.

Reply:
Added to #9583 since I forgot to do this when fixing conflicts.